core/client/fs/src/test/java/alluxio/client/file/cache/LRUCacheEvictorTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.file.cache;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Tests for the {@link LRUCacheEvictor} class.
+ */
+public final class LRUCacheEvictorTest {
+  private LRUCacheEvictor mEvictor;
+  private final PageId mFirst = new PageId(1L, 2L);
+  private final PageId mSecond = new PageId(3L, 4L);
+  private final PageId mThird = new PageId(5L, 6L);
+
+  /**
+   * Sets up the instances.
+   */
+  @Before
+  public void before() {
+    mEvictor = new LRUCacheEvictor();
+  }
+
+  @Test
```

Comment:
Shall we cover the following cases:
- `evict` before any other operations (just make sure we don't see NPE)
- `evict` after all pages are put and then deleted

Reply:
Added.